src/app/utils/date-utils.spec.ts

```diff
@@ -36,5 +36,13 @@ describe('DateUtils', () => {
 
             expect(dateUtils.getHttpFriendly(momentDate)).toEqual('01-03-2013');
         })
-    })
+    });
+
+    describe('formatDuration', () => {
+        it('should format duration properly', () => {
+            const momentDuration = moment.duration(1500, 'minutes');
```

Comment:
It would be good to add maybe 2 more test cases like when duration is less than a day and duration equals a day.